### stl of feet ########################################################
foot_mesh:
  left:  "robotis_l_leg_foot.stl"
  right: "robotis_r_leg_foot.stl"

urdf_properties:
  foot_origin_right: {x: 0, y: -0.014, z: 0.0075}
  foot_origin_left: {x: 0, y: 0.014, z: 0.0075}
  mesh_origin: {x: 0, y: 0, z: 0.0275}
  mesh_scale: {x: 0.001, y: 0.001, z: 0.001}

